tests/unit/job/swapJobHandler/swapJobHandler.spec.ts

```diff
@@ -37,7 +37,14 @@ describe('swapJobHandler', () => {
         partsData: job.parameters.partsData,
       };
 
-      const mergeTasks: AsyncGenerator<MergeTaskParameters, void, void> = (async function* () {})();
+      const mergeTasks: AsyncGenerator<
+        {
+          mergeTasksGenerator: MergeTaskParameters;
+          latestTaskIndex: { lastInsertedTaskIndex: number; zoomLevel: number };
```

Comment:
you defined a type for it : `JobResumeState` anyway reuse merge type of them both

Reply:
see comment below